tutorials/tutorial-grafana.md

```diff
@@ -0,0 +1,262 @@
+interface, which will enable you to add a panel to an existing dashboard.
+
+To proceed with our tutorial, let’s add a new visualization by clicking on the 'Choose 
+Visualization' option.
+
+At this point, you’ll have several options for different Grafana visualizations. We will 
+choose the first option, the 'Graph' visualization.
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/grafana_visualizations.png" alt="Grafana visualizations to choose from"/>
+
+There are multiple ways to configure our panel, but we will accept all the defaults 
+and create a simple 'Lines' graph.
+
+In the far left section of the Grafana user interface, select the 'Queries' tab.
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/create_grafana_query.png" alt="How to create a new Grafana query"/>
+
+Instead of using the Grafana query builder, we will edit our query directly. In the 
+view, click on the 'Edit SQL' button at the bottom.
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/edit_sql_in_grafana.png" alt="Edit custom SQL queries in Grafana"/>
+
+Before we can begin authoring our query, we also want to set the Query database to the New 
+York City taxi cab datasource we connected to earlier:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/set_datasource.png" alt="Switching data sources in Grafana"/>
+
+### Visualize metrics stored in TimescaleDB
+
+Let’s start by creating a visualization that answers the question **How many rides took place on each day?** 
+from the [Hello, Timescale!][hello-timescale] tutorial.
+
+From the tutorial, you can see the standard SQL syntax for our query:
+
+```sql
+SELECT date_trunc('day', pickup_datetime) AS day, 
+  COUNT(*) 
+FROM rides 
+GROUP BY day 
+ORDER BY day;
+```
+
+We will need to alter this query to support Grafana’s unique query syntax.
+
+#### Modifying the SELECT statement
+
+First, we will modify the `date_trunc` function to use the TimescaleDB `time_bucket` 
+function. You can consult the TimescaleDB [API Reference on time_bucket][time-bucket-reference] 
+for more information on how to use it properly.
+
+Let’s examine the `SELECT` portion of this query. First, we will bucket our results into 
+one day groupings using the `time_bucket` function. If you set the 'Format' of a Grafana 
+panel to be 'Time series', for use in Graph panel for example, then the query must return 
+a column named `time` that returns either a SQL `datetime` or any numeric datatype 
+representing a Unix epoch.
+
+So, part 1 of this new query is modified so that the output of the `time_bucket` grouping 
+is labeled `time` as Grafana requires, while part 2 is unchanged:
+
+```sql
+SELECT 
+  --1--
+  time_bucket('1 day', pickup_datetime) AS "time",
+  --2--
+  COUNT(*)
+FROM rides 
+```
+
+#### The Grafana __timeFilter function
```

Comment:
```suggestion
#### The Grafana \_\_timeFilter function
```
The double underscore cannot be left as is.  It makes the markdown parser think this is the start of a highlight section that goes throughout the rest of the page

Reply:
Either double escape, or put in backticks